monai/transforms/spatial/array.py

```diff
@@ -1245,11 +1238,15 @@ def __call__(self, spatial_size: Sequence[int]):
 
 
 class Resample(Transform):
+
+    backend = [TransformBackends.TORCH]
```

Comment:
Maybe you can add the support for numpy input for this transform directly?

Thanks.